src/fiatExchanges/FiatExchangeAmount.tsx

```diff
@@ -122,7 +123,7 @@ function FiatExchangeAmount({ route }: Props) {
     dispatch(fetchExchangeRate())
   }, [])
 
-  if (!address) {
+  if (!address && currency !== CiCoCurrency.ETH) {
```

Comment:
seems like this might need a todo to remove it right? once we have eth prices available to us?